src/include/buffer.h

```diff
@@ -881,6 +881,13 @@ namespace buffer CEPH_BUFFER_API {
     }
     uint32_t crc32c(uint32_t crc) const;
     void invalidate_crc();
+
+    // These functions return a bufferlist with a pointer to a single
+    // static buffer. They /must/ not outlive the memory they
+    // reference.
+    static list static_from_chararray(char* c, size_t l);
+    static list static_from_cstring(char* c);
+    static list static_from_string(std::string& s);
```

Comment:
ON second thought I shouldn't do that. buffer.h is one of the files included as part of the librados headers and if I add a new header dependency to librados, I think someone will kill me in my sleep.